notebooks/301-tensorflow-training-openvino/README.md

```diff
@@ -7,7 +7,7 @@
 
 In this directory, you will find two Jupyter notebooks. The first is an end-to-end deep learning training tutorial which borrows the open source code from the TensorFlow [image classification tutorial](https://www.tensorflow.org/tutorials/images/classification), demonstrating how to train the model and then convert to OpenVINO™ Intermediate Representation (IR). It leverages the [tf_flowers](https://www.tensorflow.org/datasets/catalog/tf_flowers) dataset which includes about 3,700 photos of flowers.
 
-The second notebook demonstrates how to quantize the OpenVINO IR model that was created in the first notebook. Post-training quantization speeds up inference on the trained model. The quantization is performed with the [Post Training Optimization Tool (POT)](https://docs.openvinotoolkit.org/latest/pot_README.html) from OpenVINO Toolkit. A custom dataloader and metric will be defined, and accuracy and performance will be computed for the original IR model and the quantized model on CPU and iGPU (if available).
+The second notebook demonstrates how to quantize the OpenVINO IR model that was created in the first notebook. Post-training quantization speeds up inference on the trained model. The quantization is performed with the [Post Training Optimization Tool (POT)](https://docs.openvino.ai/latest/pot_README.html) from OpenVINO Toolkit. A custom dataloader and metric will be defined, and accuracy and performance will be computed for the original IR model and the quantized model on CPU and iGPU (if available).
```

Comment:
Page not found

Reply:
@sgolebiewski-intel Thanks for the update about hyperlinks. Some URLs are not working. Please take a look at my comments. BR.

Reply:
Again, thank you for pointing that out. I have corrected them, and they seem to be working now.

Reply:
Hi there, it's seems the build fails because of sth else. I've tracked the logs. If I'm not mistaken, the culprit is:
test_211-speech-to-text.ipynb